eng/Packages.Data.props

```diff
@@ -186,7 +186,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Azure.AI.OpenAI'))">
-    <PackageReference Update="OpenAI" Version="2.1.0-beta.2" />
+    <PackageReference Update="OpenAI" Version="2.1.*" />
```

Comment:
Please make this a specific version.  We do not allow for wild cards, as we want to ensure that we have deterministic repeatable builds - especially when building from release tags for hotfixes.